django/core/files/move.py

```diff
@@ -66,7 +66,13 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
+        # This will happen with copystat if the destination is a CIFS mount
+        if getattr(e, 'errno', 0) != 1:
```

Comment:
Thanks @felixxm and @claudep.  @peterfarrell and I were just chatting and our plan tomorrow is to change to errno.EPERM and provide tests around this.